spec.html

```diff
@@ -17816,16 +17799,16 @@
       <emu-alg>
         1. Return NormalCompletion(~empty~).
       </emu-alg>
-      <emu-grammar>FormalParameterList : FormalsList `,` FunctionRestParameter</emu-grammar>
+      <emu-grammar>FormalParameters : FormalParameterList `,` FunctionRestParameter</emu-grammar>
       <emu-alg>
-        1. Let _restIndex_ be the result of performing IteratorBindingInitialization for |FormalsList| using _iteratorRecord_ and _environment_ as the arguments.
+        1. Let _restIndex_ be the result of performing IteratorBindingInitialization for |FormalParameterList| using _iteratorRecord_ and _environment_ as the arguments.
         1. ReturnIfAbrupt(_restIndex_).
         1. Return the result of performing IteratorBindingInitialization for |FunctionRestParameter| using _iteratorRecord_ and _environment_ as the arguments.
       </emu-alg>
-      <emu-grammar>FormalsList : FormalsList `,` FormalParameter</emu-grammar>
+      <emu-grammar>FormalParameterList : FormalParameterList `,` FormalParameter</emu-grammar>
       <emu-alg>
-        1. Let _status_ be the result of performing IteratorBindingInitialization for |FormalsList| using _iteratorRecord_ and _environment_ as the arguments.
-        1. ReturnIfAbrupt(_status_).
+        1. Let _restIndex_ be the result of performing IteratorBindingInitialization for |FormalParameterList| using _iteratorRecord_ and _environment_ as the arguments.
+        1. ReturnIfAbrupt(_restIndex_).
```

Comment:
this change is not wrong but neither necessary. _status_ would be fine.

No need to revert, but is there any specific reason for this change?